ansible/provision.yml

```diff
@@ -6,6 +6,11 @@
       - lamp
 
   tasks:
+    - name: set git config values
```

Comment:
Seems to be an issue with different indention for this task "set git config values" and the task below "copy profile files".

Reply:
. . . which has now been done.